ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -3511,13 +3512,18 @@ static void ggml_vk_load_shaders(vk_device& device) {
     // the number of rows computed per shader depends on GPU model and quant
     uint32_t rm_stdq = 1;
     uint32_t rm_kq = 2;
+    uint32_t rm_stdq_int = 1;
```

Comment:
My WSL build is still using an old glslc without int dot support, and gets these errors:

```

/mnt/c/github/jeffbolznv/llama.cpp/ggml/src/ggml-vulkan/ggml-vulkan.cpp: In function ‘void ggml_vk_load_shaders(vk_device&)’:
/mnt/c/github/jeffbolznv/llama.cpp/ggml/src/ggml-vulkan/ggml-vulkan.cpp:3515:14: error: variable ‘rm_stdq_int’ set but not used [-Werror=unused-but-set-variable]
 3515 |     uint32_t rm_stdq_int = 1;
      |              ^~~~~~~~~~~
/mnt/c/github/jeffbolznv/llama.cpp/ggml/src/ggml-vulkan/ggml-vulkan.cpp:3516:14: error: unused variable ‘rm_kq_int’ [-Werror=unused-variable]
 3516 |     uint32_t rm_kq_int = 1;
      |              ^~~~~~~~~
cc1plus: all warnings being treated as errors
```